packages/beacon-node/src/api/impl/beacon/state/index.ts

```diff
@@ -254,8 +254,28 @@ export function getBeaconStateApi({
       }
 
       const epoch = filters.epoch ?? computeEpochAtSlot(state.slot);
+      const stateEpoch = computeEpochAtSlot(state.slot);
+      if (filters.slot !== undefined) {
+        const epochStartSlot = computeStartSlotAtEpoch(epoch);
+        const epochEndSlot = epochStartSlot + 32 - 1;
```

Comment:
this will break with minimal preset as an epoch only has 8 slots, always need to use `SLOTS_PER_EPOCH`